packages/neuron-ui/src/components/Transaction/index.tsx

```diff
@@ -27,25 +41,42 @@ const Transaction = () => {
           name: t('transaction.index'),
           minWidth: 60,
           maxWidth: 60,
-          onRender: (_item?: any, index?: number) => {
+          onRender: (_input?: State.DetailedInput, index?: number) => {
             if (undefined !== index) {
               return index
             }
             return null
           },
         },
         {
-          key: 'outPointCell',
-          name: 'OutPoint Cell',
-          minWidth: 150,
-          maxWidth: 600,
-          onRender: (item: any) => {
-            const text = item.previousOutput ? `${item.previousOutput.txHash}[${item.previousOutput.index}]` : 'none'
-            return (
-              <span title={text} className="textOverflow">
-                {text}
-              </span>
-            )
+          key: 'address',
+          name: t('transaction.address'),
+          minWidth: 200,
+          maxWidth: 500,
+          onRender: (input?: State.DetailedInput, _index?: number, column?: IColumn) => {
+            if (!input) {
+              return null
+            }
+            if (!input.lock) {
+              return t('transaction.cell-from-coinbase')
```

Comment:
`cellbase` or `coinbase` ?

Reply:
fixed in https://github.com/nervosnetwork/neuron/pull/987/commits/27df45391736ff96612ab85f898784dce7da34e3